.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
```

Comment:
I'm not sure that we want utf-8 but I guess that everything can handle it gracefully these days.

Reply:
This is what they have so far https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#charset
We could also just use latin1 ?

Reply:
done